virtual_rainforest/main.py

```diff
@@ -190,9 +190,10 @@ def extract_timing_details(
 
 
 def vr_run(
-    cfg_paths: Optional[Union[str, Path, Sequence[Union[str, Path]]]] = None,
-    cfg_string: Optional[str] = None,
+    cfg_paths: Union[str, Path, Sequence[Union[str, Path]]] = [],
+    cfg_strings: Union[str, list[str]] = [],
```

Comment:
I'm not fully sure of the use case for this. It is just to simplify testing?

Reply:
Yes. All over the place we have snippets of code where we want to create a particular config. This allows us to do this without having to mock files. It just makes sense to be able to create a config without using a file.

We don't expect to do this from the command line,  so this argument isn't used in `vr_run_cli`.